### PR TITLE
LPS-101720

### PR DESCRIPTION
**Pull request by brianchandotcom**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
Please read my comment in https://github.com/brianchandotcom/liferay-portal/pull/83704 thank you

**Comment by csierra** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-email-otp-web/src/main/java/com/liferay/multi/factor/authentication/email/otp/web/internal/portlet/action/SendMFAEmailOTPMVCResourceCommand.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public boolean serveResource(
 
 			if (userId == null) {
 				if (_log.isWarnEnabled()) {
-					_log.warn("User id is not in the session");
```
Comment (csierra) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-email-otp-web/src/main/java/com/liferay/multi/factor/authentication/email/otp/web/internal/portlet/action/SendMFAEmailOTPMVCResourceCommand.java]: hey @brianchandotcom, I copied this snippet from the portal: 
https://github.com/liferay/liferay-portal/blob/b3173da81b62430f7150b695d5e944b224279580/portal-impl/src/com/liferay/portal/servlet/PortalSessionDestroyer.java#L67

**Comment by csierra** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/83750. [See changes here.](https://github.com/csierra/liferay-portal/compare/12cf0adbcfb5~3...12cf0adbcfb5) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by csierra** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/83751. [See changes here.](https://github.com/csierra/liferay-portal/compare/e79871ddbdab~3...e79871ddbdab) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by csierra** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/83752. [See changes here.](https://github.com/csierra/liferay-portal/compare/d84a6b1f3e59~3...d84a6b1f3e59) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by csierra** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/83789. [See changes here.](https://github.com/csierra/liferay-portal/compare/91ff949c87de~6...91ff949c87de) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**